src/main/java/edu/cornell/kfs/pdp/service/impl/CuPaymentFileValidationServiceImpl.java

```diff
@@ -216,11 +221,34 @@ protected void checkPaymentGroupPropertyMaxLength(PaymentGroup paymentGroup, Mes
         }
     }
 
+    // This is an altered copy of checkPaymentGroupPropertyMaxLength() that operates on Payment Details instead.
```

Comment:
do we need this comment?

Reply:
I don't this comment is really adding much value, so I'll remove it. Thanks.